src/modules/test/recipes/CommandRunnerTests.json

```diff
@@ -1,34 +1,98 @@
 [
-    {
-        "ComponentName": "CommandRunner",
-        "ObjectName": "commandStatus",
-        "Desired": false,
-        "ExpectedResult": 0,
-        "WaitSeconds": 0
-    },
-    {
-        "ComponentName": "CommandRunner",
-        "ObjectName": "commandArguments",
-        "Desired": true,
-        "Payload": "{\"commandId\": \"helloworld\", \"arguments\": \"echo Hello World!\", \"timeout\": 0, \"singleLineTextResult\": true, \"action\": 100, }",
-        "PayloadSizeBytes": 0,
-        "ExpectedResult": 22,
-        "WaitSeconds": 0
-    },
     {
         "ComponentName": "CommandRunner",
         "ObjectName": "commandArguments",
         "Desired": true,
-        "Payload": "{\"commandId\": \"test1\", \"arguments\": \"echo Hello World\", \"timeout\": 0, \"singleLineTextResult\": true, \"action\": 3}",
-        "PayloadSizeBytes": 123,
+        "Payload": "{\"commandId\": \"RmCache\", \"arguments\": \"sudo rm /etc/osconfig/osconfig_commandrunner.cache\", \"timeout\": 0, \"singleLineTextResult\": true, \"action\": 3}",
```

Comment:
this will fail on CI, no sudo in the container environments and if the file is not present, rm returns an error. I would recommend using `rm /etc/osconfig/osconfig_commandrunner.cache || true` to ensure it works in all environments